services/ui-src/src/components/pages/Dashboard/DashboardPage.tsx

```diff
@@ -180,6 +181,8 @@ export const DashboardPage = ({ reportType }: Props) => {
     onClose: addEditReportModalOnCloseHandler,
   } = useDisclosure();
 
+  const fullStateName = States[userState as keyof typeof States];
```

Comment:
Question: Should we use `activeState` over `userState` here, so that admin users will see the name of the state whose reports they are viewing?

Reply:
Oooh, good call out. I think we should use `activeState` so that admin users see the name of the state corresponding to the report. I will revise!